tools/@aws-cdk/cdk-build-tools/bin/cdk-lint.ts

```diff
@@ -19,11 +20,17 @@ async function main() {
 
   const options = cdkBuildOptions();
 
-  await lintCurrentPackage(options, { eslint: args.eslint, fix: args.fix });
+  await lintCurrentPackage(options, timers, { eslint: args.eslint, fix: args.fix });
 }
 
+const timers = new Timers();
+const buildTimer = timers.start('Total time');
+
 main().catch(e => {
   process.stderr.write(`${e.toString()}\n`);
   process.stderr.write('Linting failed.\n');
   process.exit(1);
+}).finally(() => {
+  buildTimer.end();
+  process.stdout.write(`Build times for ${currentPackageJson().name}: ${timers.display()}\n`);
```

Comment:
Should this be something like this?
```suggestion
  process.stdout.write(`Lint times for ${currentPackageJson().name}: ${timers.display()}\n`);
```